docker-compose.yml

```diff
@@ -13,6 +14,9 @@ services:
       dockerfile: Dockerfile
     ports:
       - "8000:8000"
+    restart: always
+    environment:
+      SPRING_DATA_MONGODB_HOST=mongo-db:
```

Comment:
Nice! does this mean that if the application crashes on its own, it will immediately spin up another container? If I remember this setting, it excludes manual stops.

Reply:
Yep! Thats exactly what it does